src/eval-frame/actions/language-actions.js

```diff
@@ -50,6 +50,8 @@ function loadLanguagePlugin(pluginData, historyId, cell, dispatch) {
         // see the following for asynchronous loading of scripts from strings:
         // https://developer.mozilla.org/en-US/docs/Games/Techniques/Async_scripts
 
+        window.languagePluginUrl = url
```

Comment:
Hmm, yeah, this feels a little icky to me :/ Could we not modify the responseText and pass the value in there? If not, I guess this is ok.